bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -1043,8 +1069,8 @@ private void notifyGroupSceneUpdate(List<Scene> scenes) {
 
         String ip = hueBridgeConfig.getIpAddress();
```

Comment:
As detected GitHub actions, there should be a problem here as you removed the method `getIpAddress` from the config class.